frontend/packages/knative-plugin/src/components/add/event-sources/__tests__/EventSourcesSelector.spec.tsx

```diff
@@ -48,15 +46,4 @@ describe('EventSourcesSelector', () => {
     wrapper = shallow(<EventSourcesSelector eventSourceList={eventSourceList} />);
     expect(wrapper.find(ItemSelectorField).props().loadingItems).toBe(false);
   });
-
-  it('should call getEventSourceData onSelect', () => {
-    const spyGetEventSourceData = jest.spyOn(sourceUtils, 'getEventSourceData');
-    const spyKebabCase = jest.spyOn(lodash, 'kebabCase');
-    wrapper
-      .find(ItemSelectorField)
-      .props()
-      .onSelect('ApiServerSource');
-    expect(spyGetEventSourceData).toHaveBeenCalledWith('apiserversource');
-    expect(spyKebabCase).toHaveBeenCalledWith('ApiServerSource');
-  });
```

Comment:
why removed this spec , i would expect it to be modified based on new changes

Reply:
the spec was removed when the code for `handleChange` was moved out of the component.
reintroduced the test